client/sites/components/site-preview-pane/preview-pane-header-buttons.scss

```diff
@@ -1,4 +1,6 @@
 .item-preview__sync-dropdown {
+	width: min-content;
```

Comment:
Required to avoid the overriding that the editor applies 
<img width="2252" height="2208" alt="CleanShot 2025-08-05 at 18 15 10@2x" src="https://github.com/user-attachments/assets/d9ba744a-0fa7-4a2f-a62f-f06f3c1e0011" />